cli_tools/gce_vm_image_import/importer/importer_test.go

```diff
@@ -150,111 +145,32 @@ func TestFlagsDataDiskAndOSFlagsBothProvided(t *testing.T) {
 	assertErrorOnValidate("Expected error for both os and data_disk set at the same time", t)
 }
 
-func TestFlagsSourceFileOrSourceImageNotProvided(t *testing.T) {
-	resetArgs()
-	sourceFile = ""
-	sourceImage = ""
-	dataDisk = false
-	assertErrorOnValidate("Expected error for missing source_file or source_image flag", t)
-}
-
-func TestFlagsSourceFileAndSourceImageBothProvided(t *testing.T) {
-	resetArgs()
-	sourceFile = "gs://source_bucket/source_file"
-	dataDisk = false
-	assertErrorOnValidate("Expected error for both source_file and source_image flags set", t)
-}
-
 func TestFlagsSourceFile(t *testing.T) {
 	resetArgs()
-	sourceFile = "gs://source_bucket/source_file"
-	sourceImage = ""
 	dataDisk = false
 
-	if _, _, _, err := validateAndParseFlags(clientID, imageName, sourceFile, sourceImage, dataDisk,
-		osID, customTranWorkflow, labels); err != nil {
-		t.Errorf("Unexpected error: %v", err)
-	}
-}
-
-func TestFlagSourceFileEmpty(t *testing.T) {
```

Comment:
These tests are moved to source_test.go which makes sense, but I'm not sure current test setup will ensure that initAndValidateSource is called in Run(). Will tests fail if initAndValidateSource() call is deleted by mistake?

Reply:
Which test actually verifies that initAndValidateSource() is called from the importer?